src/sql/sqlops.proposed.d.ts

```diff
@@ -844,6 +844,22 @@ declare module 'sqlops' {
 		initializeModel<T extends Component>(root: T): Thenable<void>;
 	}
 
+	export namespace adsWindowsSupport {
```

Comment:
Why do we need to add this to proposed? This is if core is providing apis to extensions. I don't see any changes in core that are providing new functionality to extensions.